yb-voyager/src/srcdb/source.go

```diff
@@ -151,12 +151,21 @@ func generateSSLQueryStringIfNotExists(s *Source) string {
 }
 
 //go:embed data/sample-ora2pg.conf
-var SampleOra2pgConfigFile string
+var Ora2pgConfigFile string
 
 func (source *Source) PopulateOra2pgConfigFile(configFilePath string) {
 	sourceDSN := source.getSourceDSN()
-
-	lines := strings.Split(string(SampleOra2pgConfigFile), "\n")
+	var SampleOra2pgConfigFile string
+	if strings.HasSuffix(configFilePath, "base_ora2pg.conf") {
```

Comment:
You can use `filepath.Base()` to extract the file name from file path.
Prefer either `-` or `_` for both conf files